### DISCUSSION TITLE
Welcome everybody!

### DISCUSSION THREAD

**Discussion by timoschlueter** [timoschlueter/nightscout-librelink-up#103] [Announcements]
Since this repository has grown a lot over the last year i thought it would be time to start using GitHub discussions to share knowledge between users.

Feel free to ask for help here. If there are software related errors, please submit an issue. For general help please start or engage in a topic here. :)

**Comment by Loallisk** [comment]
Hey, has anybody been able to get this to work with a northflank-nightscout setup?

**Reply by Zurnl** [reply]
Hi,i managed to get it work on Northflank by following this instruction -> https://www.beausteward.com/posts/2022/libre-2-to-nightscout/

**Comment by Zurnl** [comment]
Hey, thank you for creating this project. 
I've been running Nightscout for my son for many years. After Heroku stopped offering free tier i moved to Fly.
We had a nice working setup: Freestyle Libre2->MiaMiao3->xDrp4iO5 on old iPad->Nightscout on fly.io
Then he got the much waited Libre3 and trashed our setup. 
This project hopefully brings back NightScout nighttime monitoring. I maybe have to move back to heroku or try docker path.

**Comment by mduiella** [comment]
Hello,
My wife is switching from FS2 to FS3, and we would like to keep using XDrip4iOS. For this, we would like to have glucose readings uploaded into NightScout from LibreLinkUp, so that XDrip4iOS can be set as follower.
The problem is that we switched from Heroku to Nightscout on Google cloud (with Railway in between), and I don't know how to deploy this on Google cloud...
Any idea how this could be done? (I have asked already to the developers of NS on Google Cloud, but they don't have Libre and couldn't really help...)
Many thanks in advance!
Matteo

**Comment by robster7674** [comment]
@Loallisk yesterday I added a container to my NorthFlank NigthScout deployment, and it has been running with 1-minute intervals for almost 24 hours now. I'm using FreeStyle Libre 2 with latest app on Android to get 1-minute readings. Best of all, it's free still!

@mduiella I suggest you use NorthFlank. Pretty easy to set up and great support, even on the free plan.

@Zurnl I also moved to Fly, but that ended as well. I am now on NorthFlank, loving it.

Edit: btw., I'm using Nightscout Menu Bar app from Mac Appstore to get readings in my Mac's menubar. Latest version even has a graph.
